pkg/trace/obfuscate/sql.go

```diff
@@ -351,7 +351,7 @@ func attemptObfuscationWithOptions(tokenizer *SQLTokenizer, opts SQLOptions) (*O
 		if token, buff, err = discard.Filter(token, lastToken, buff); err != nil {
 			return nil, err
 		}
-		if storeTableNames || quantizeTableNames {
+		if storeTableNames || replaceDigits {
```

Comment:
I think you can now remove the `|| replaceDigits` part, right? Table names will simply be identified as `ID` and it'll work fine.

Reply:
Updated: https://github.com/DataDog/datadog-agent/pull/8860/commits/bec8b338f1048c08f711e9197a2568ef723c7138